package/MDAnalysis/core/universe.py

```diff
@@ -298,6 +298,12 @@ def __init__(self, *args, **kwargs):
         # Universes are anchors by default
         self.is_anchor = kwargs.get('is_anchor', True)
 
+    def copy(self):
```

Comment:
it might be possible to have `shallow=True` which would allow them to share a `topology`.  This would be beneficial when you're doing something like a MSD calculation, so you just want two independent trajectory objects to iterate with to compare frames.

Reply:
Do you have a test for this behavior?

Reply:
https://github.com/MDAnalysis/mdanalysis/blob/4aaff902c20d7638391f22a832f42b23a4e95991/testsuite/MDAnalysisTests/core/test_copying.py#L183